src/components/savedLocations/components/SaveLocationById.js

```diff
@@ -1,18 +1,29 @@
-import React, { useEffect } from 'react';
-import { useSelector, useDispatch } from 'react-redux';
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
 import { Card, Row, Col, Button, Icon, Typography } from 'antd';
-import { getSavedLocationsByID } from '../actions';
 import LocationList from './LocationList';
 
 const SaveLocationById = ({ match, history }) => {
     const { params } = match;
     const { id: savedLocationId } = params;
-    const { savedLocationById } = useSelector(state => state.savedLocation);
-    const { title } = savedLocationById;
-    const dispatch = useDispatch();
+    const { savedLocations } = useSelector(state => state.savedLocation);
+    const { locations } = useSelector(state => state.location);
+    const [newSavedLocation, setnewSavedLocation] = useState({});
```

Comment:
since you are using camel case, I believe it should be `setNewSavedLocation` and not `setnewSavedLocation`